_includes/navigation.html

```diff
@@ -15,50 +15,56 @@ <h2 class="pldoc-heading pldoc-nav-title hd-6">Elements</h2>
     </ul>
 </nav>
 
-<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation">
-    <h2 class="pldoc-heading pldoc-nav-title hd-6">Documentation</h2>
-    <ul class="pldoc-list nav-list list-styleguides">
+<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation" role="navigation" aria-label="Documentation">
+    <h2 class="pldoc-heading pldoc-nav-title hd-6" id="heading-documentation">Documentation</h2>
+    <ul class="pldoc-list nav-list list-styleguides" aria-labelledby="heading-documentation">
         <li class="nav-list-item">
             <a class="pldoc-link" rel="external" href="https://github.com/edx/ux-pattern-library/wiki/Styleguide:-General">
                 <span class="pldoc-icon" aria-hidden="true">&#65514;</span>
                 General Styleguide
+                <span class="sr-only">Link opens in a new window</span>
```

Comment:
Hmmm, is this the general best practice for new tab/window links? If so, please change the copy and wrap this text in parenthesis so the link copy reads "Link Text (this link in a new window)"